Jenkinsfile

```diff
@@ -334,6 +336,9 @@ pipeline {
                                 # integration-cli second set
                                 TEST_INTEGRATION_DEST=3 CONTAINER_NAME=${CONTAINER_NAME}-3 TEST_SKIP_INTEGRATION=1 TESTFLAGS="-test.run Test(DockerSwarmSuite|DockerDaemonSuite|DockerExternalVolumeSuite)/" run_tests &
 
+                                # Rootless integration (no CLI)
+                                TEST_INTEGRATION_DEST=4 CONTAINER_NAME=${CONTAINER_NAME}-4 TEST_SKIP_INTEGRATION_CLI=1 DOCKER_EXPERIMENTAL=1 DOCKER_ROOTLESS=1 run_tests &
```

Comment:
Perhaps instead we should just create a separate stage for this in the jenkinsfile, so that we can more flexibly enable/disable things without tripping up the main integration tests 🤔

Reply:
added a separate stage, PTAL